scipy/cluster/hierarchy.py

```diff
@@ -3177,10 +3177,19 @@ def llf(id):
                     return str(id)
                 else:
                     return '[%d %d %1.2f]' % (id, count, R[n-id,3])
+
             # The text for the leaf nodes is going to be big so force
             # a rotation of 90 degrees.
             dendrogram(Z, leaf_label_func=llf, leaf_rotation=90)
 
+            # leaf_label_func can also be used together with ``truncate_mode`` parameter,
+            # in which case you will get your leaves labeled after truncation:
+            # dendrogram(Z, leaf_label_func=llf, leaf_rotation=90,\
+                    truncate_mode='level', p=5)
```

Comment:
These last two lines render a bit odd, see https://25402-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.cluster.hierarchy.dendrogram.html#scipy.cluster.hierarchy.dendrogram.

I think it would be useful to typeset them as code, so can you remove the `#`, remove `\`, and align `Z` and `truncate`? Also please leave only one blank line below.

Reply:
My bad, I did not test it, I'm sorry. Fixed as requested, and also changed `p=5` to `p=2`. I hope someone will make use of it